### PR TITLE
otelcolconvert: support converting servicegraph connector

### LINKED ISSUE #6443: Support converting `servicegraphconnector`

### PR DESCRIPTION
**Pull request by tpaschalis**
Closes #6443

### CODE REVIEW

File: internal/converter/internal/otelcolconvert/testdata/servicegraph.river [hunk 1]
```diff
@@ -0,0 +1,33 @@
+otelcol.receiver.otlp "default" {
+	grpc { }
+
+	http { }
+
+	output {
+		metrics = [otelcol.exporter.otlp.default.input]
```
Comment (erikbaranowski) [inline internal/converter/internal/otelcolconvert/testdata/servicegraph.river:7]: Is this correct to have the otlp receiver send to the otlp exporter directly? I'm trying to understand the concept of connectors in Otel so let me know.

Comment (rfratto) [reply internal/converter/internal/otelcolconvert/testdata/servicegraph.river:7]: Ooh, I think this is a bug in the framework. The otlp receiver is not configured in any case to send metrics to the otlp exporter, so no, this isn't right. Nice catch.

Comment (tpaschalis) [reply internal/converter/internal/otelcolconvert/testdata/servicegraph.river:7]: Whoops, will take a look, thanks for not glancing over it.

Comment (erikbaranowski) [reply internal/converter/internal/otelcolconvert/testdata/servicegraph.river]: This should be all fixed now

**Review by erikbaranowski — APPROVED** [review]

**Pull request merged**